### ISSUE TITLE
Unhandled Promise Rejection

### ISSUE THREAD

**Issue by shacker** [jrief/django-admin-sortable2#313]
admin-sortable2 2.0.4 in my project works fine on localhost. But deployed to a server, the second I touch a drag handle, the console shows this error:

<img width="683" alt="CleanShot 2022-05-24 at 22 58 55@2x" src="https://user-images.githubusercontent.com/102694/170190746-9334964b-396c-4a9b-8574-6a19f340362d.png">

And ordering changes are not saved. Whatever `t` is, it's undefined! I've confirmed that `/static/adminsortable2/js/adminsortable2.min.js` is definitely loading, with a 200. 

Is this a bug, or could I have misconfigured something?

**Comment by shacker** [comment]
No one else is encountering this?  Any theories?

**Comment by Eccleshall** [comment]
@shacker This appears to be an issue with the minified version (adminsortable2.min.js) the un-minified version works as expected.

As a temp work around add below code to your admin class that inherits `SortableAdminMixin` which will remove **min** from the file path name which is applied when DEBUG = False  as per [SortableAdminBase - adminsortable2/admin.py line 66.](https://github.com/jrief/django-admin-sortable2/blob/master/adminsortable2/admin.py#L66)


```
    @property
    def media(self):
        css = {"all": ["adminsortable2/css/sortable.css"]}
        js = ["adminsortable2/js/adminsortable2.js"]
        return super().media + widgets.Media(css=css, js=js)

```

**Comment by jrief** [comment]
@Eccleshall thanks for looking at this problem. What do you think, shall I refer to the unminimized version only?

**Comment by shacker** [comment]
@Eccleshall @jrief Thanks for the confirmation! I can live with the workaround until the next release. Cheers!

**Comment by christianwgd** [comment]
Simply copying the unminified version also works fine and doesn't need code changes. When a release with fix is available manage.py collectstatic will switch back to the minified version.

**Comment by jrief** [comment]
Minimizing the JS file has been built into the deployment script. I hope this will fix it.

**Comment by christianwgd** [comment]
Sorry to say, updated to 2.1.2, but this issue still appears. After copying the unminified version to the minified the error is gone. Before that i deleted the browser cache, reinstalled the library, "force updated" the site and so on.

**Comment by jrief** [comment]
Fixed in version 2.1.3 by using terser instead of uglify.
